README-tmp.md

```diff
@@ -0,0 +1,379 @@
+# 5.9.5 Configuring EFS Cross Account
+
+## 5.9.5.1 Requirements
+
+- The EFS CSI Operator must be installed prior to beginning this procedure. Follow the _Installing the AWS EFS CSI Driver Operator_ procedure.
+- Both the Red Hat OpenShift Service on AWS cluster and EFS file system must be located in the same AWS region.
+- Ensure that the two VPCs referenced in this guide utilize different network CIDR ranges.
+
+## 5.9.5.2 Prerequisites
+
+- AWS Account A: Contains a Red Hat OpenShift Cluster 4.16 or later, deployed within a VPC
+- AWS Account B: Contains a VPC (including subnets, route tables, and network connectivity). The EFS filesystem will be created in this VPC during this procedure
+- OpenShift CLI (`oc`)
+- AWS CLI
+- `jq` command-line JSON processor
+
+## 5.9.5.3 Environment Setup
+
+1. Configure environment variables that will be referenced throughout this guide:
+
+```bash
+export CLUSTER_NAME="<CLUSTER_NAME>" #Cluster name of choice
+export AWS_REGION="<AWS_REGION>" #AWS Region of choice
+export AWS_ACCOUNT_A_ID="<ACCOUNT_A_ID>" #AWS Account B ID
+export AWS_ACCOUNT_B_ID="<ACCOUNT_B_ID>" #AWS Account B ID 
+export AWS_ACCOUNT_A_VPC_CIDR="<VPC_A_CIDR>" #CIDR range of VPC in Account A
+export AWS_ACCOUNT_B_VPC_CIDR="<VPC_B_CIDR>" #CIDR range of VPC in Account B
+export AWS_ACCOUNT_A_VPC_ID="<VPC_A_ID>" #VPC ID in Account A (cluster)
+export AWS_ACCOUNT_B_VPC_ID="<VPC_B_ID>" #VPC ID in Account B (EFS cross account)
+export SCRATCH_DIR="<WORKING_DIRECTORY>" #Any writeable directory of choice, used to store temporary files
+export AWS_PAGER="" #Make AWS CLI output everything directly to stdout
+```
+
+2. Create the working directory:
+```bash
+mkdir -p $SCRATCH_DIR
+```
+
+3. Verify cluster connectivity using the `oc` command:
+
+```bash
+oc whoami
+```
+
+4. Configure AWS CLI profiles as environment variables for account switching:
+
+```bash
+export AWS_ACCOUNT_A="<ACCOUNT_A_NAME>"
+export AWS_ACCOUNT_B="<ACCOUNT_B_NAME>"
+```
+
+5. Ensure that your AWS CLI is configured with JSON output format as the default for both accounts:
```

Comment:
```suggestion
5. Ensure that your AWS CLI is configured with JSON output format as the default for both accounts(the output format could be override by `export AWS_DEFAULT_OUTPUT="json"`):
```
Maybe better.

Reply:
Yes this is one of the methods of changing the output format, however I intentionally avoided explaining it here since we would be documenting AWS here, which is not desired. For this purposes I've added a link in the paragraph below to official AWS doc (which also explains AWS_DEFAULT_OUTPUT var): https://github.com/openshift/csi-operator/pull/401/files#diff-5c4f1f33bb4c41d6c3e57b67e95dc097ebf581e3fdc2f5bbf848f15135179035R61